lib/matplotlib/cbook.py

```diff
@@ -2327,3 +2327,95 @@ def get_instancemethod(self):
 else:
     def _putmask(a, mask, values):
         return np.copyto(a, values, where=mask)
+
+class OrderedDict(dict):
```

Comment:
The ref is in the official doc https://docs.python.org/2/library/collections.html#collections.OrderedDict